.github/workflows/ci.yml

```diff
@@ -100,6 +111,16 @@ jobs:
                -e "/pytorch-gpu/c\  - pytorch-cpu ${{ matrix.pytorch }}" \
                environment.yml
 
+    - name: Prepare dependencies (Mac)
+      if: ${{ contains(matrix.os, 'macos') }}
+      run: |
+        sed -i '' -e "s/- cudatoolkit/# - cudatoolkit/" \
```

Comment:
It really looks like you are trying to install cudatoolkit here in OSX.
Not sure how to make it more obvious that you are commenting out the line.
Perhaps `sed -i '' "/- cudatoolkit/d"`

Reply:
hmm yeah I see what you mean! this should be swapped over in 4014ff4